OpenRA.Mods.Common/Scripting/Properties/ConditionProperties.cs

```diff
@@ -48,10 +48,11 @@ public void RevokeCondition(int token)
 			conditionManager.RevokeCondition(Self, token);
 		}
 
-		[Desc("Check whether this actor accepts a specific external condition.")]
+		[Desc("Check whether this actor accepts a specific external condition.",
+		      "The timed parameter does nothing now and is deprecated.")]
```

Comment:
We haven't shipped this method in a release yet, so backwards compatibility isn't a problem.  Please just remove the parameter.